src/OneWireHub.cpp

```diff
@@ -420,6 +420,7 @@ bool OneWireHub::recvAndProcessCmd(void)
 
             slave_selected = nullptr;
             searchIDTree();
+            slave_selected->duty(this);
```

Comment:
why is that needed? gonna remove this line as it is not default OW-behavior.

Reply:
Oh man, didn't see that before! Sorry, you are totally right, some ICs allow that. Will implement IC-dependent option for that! But will have to disable it for now as it is messing with the other ICs.